source/database/mssql/scripts/dbscripts/PSP_PIMS_S87_00/Alter Up/154_DML_PIMS_LEASE_STAKEHOLDER_TYPE_Alter_Up.sql

```diff
@@ -0,0 +1,53 @@
+IF NOT EXISTS (
```

Comment:
It looks like Doug did a migration of the old values in the alter up, but forgot to add these new values. They are present in the BUILD.